README.md

```diff
@@ -26,7 +26,7 @@ effective**:
 
 Compilers using Binaryen include:
 
- * [`AssemblyScript`](https://github.com/AssemblyScript/assemblyscript) which compiles a subset of TypeScript to WebAssembly
+ * [`AssemblyScript`](https://github.com/AssemblyScript/assemblyscript) which compiles a strict variant of TypeScript to WebAssembly
```

Comment:
Is there a difference between a "variant" and a "strict variant?"

Reply:
Hmm, good question. I find it hard to convey what AS is while still keeping the explanation short. What this tries to say is that AS is a variant (not purely a subset, not really a superset) of TS, but in a way that it is certainly stricter than TS, as a hint that just compiling idiomatic TS with it is not a realistic expectation at this point. Does that make sense? :)

Reply:
Yes, thanks. I was wondering if there was some technical distinction here like between "subset" and "strict subset," but it sounds like this is meant to more informally describe the variant. Perhaps using "stricter" instead of "strict" would help clarify? Either way, LGTM :)

Reply:
Now that you mention it, yeah, that may indeed be a bit confusing. "Stricter" sounds good :)